llpc/lower/llpcSpirvProcessGpuRtLibrary.cpp

```diff
@@ -136,13 +137,19 @@ SpirvProcessGpuRtLibrary::LibraryFunctionTable::LibraryFunctionTable() {
 //
 // @param func : The function to process
 void SpirvProcessGpuRtLibrary::processLibraryFunction(Function *&func) {
-  auto &funcTable = LibraryFunctionTable::get().m_libFuncPtrs;
+  auto &gpurtFuncTable = LibraryFunctionTable::get().m_libFuncPtrs;
+  auto &commonFuncTable = InternalLibraryIntrinsicUtil::LibraryFunctionTable::get().m_libFuncPtrs;
 
-  auto funcIt = funcTable.find(func->getName());
-  if (funcIt != funcTable.end()) {
-    auto funcPtr = funcIt->second;
+  auto gpurtFuncIt = gpurtFuncTable.find(func->getName());
+  auto commonFuncIt = commonFuncTable.find(func->getName());
+  if (gpurtFuncIt != gpurtFuncTable.end()) {
+    auto funcPtr = gpurtFuncIt->second;
     m_builder->SetInsertPoint(clearBlock(func));
     (this->*funcPtr)(func);
+  } else if (commonFuncIt != commonFuncTable.end()) {
```

Comment:
Maybe instead: early `return` in the first if, and move all the commonFuncTable handling (including the call to `find`) to after the first if?